content/docs/practices/naming.md

```diff
@@ -84,4 +84,4 @@ The list is not exhaustive.
 | Electric current | amperes | |
 | Energy | joules | |
 | Mass   | grams | _grams_ is preferred over _kilograms_ to avoid issues with the _kilo_ prefix. |
- 
+| Power  | watts | Prefer exporting a counter of joules, then `rate(joules[5m])` gives you power in Watts. |
```

Comment:
You added a base unit since the last push, watts still aren't the base unit. If we were to put anything in here it would be joules.

Also this should go beside the energy entry - like we do with bits&bytes.